### DISCUSSION TITLE
git trailers in log

### DISCUSSION THREAD

**Discussion by Coobaha** [jj-vcs/jj#3534] [Q&A]
Hi, is it possible to extract the git trailer information from the description? Do you know if it can be visible in the log? I assume 

It would be nice to have built-in support to extract them easily; it may already be possible with templates? 🙏 

- https://git-scm.com/docs/git-interpret-trailers
- https://git-scm.com/docs/pretty-formats#Documentation/pretty-formats.txt-emtrailersltoptionsgtem

**Comment by yuja** [comment]
There's no special support for trailer text, so you'll need to extract it from `description`.

#2845 adds utility functions, and I think some of them can be exposed as template functions.
